.pre-commit-config.yaml

```diff
@@ -9,3 +9,4 @@ repos:
     rev: 3.8.3
     hooks:
     -   id: flake8
+        args: ['--ignore=E702,E722']
```

Comment:
Multiple statements on one line (semicolon) (E702)
Do not use bare except, specify exception instead (E722)
The usage of those two seems to be always intentional, this is why I ignore them globally.
Let me know if you want to ignore them in-place.

Reply:
`ignore` is almost always wrong -- it's what's triggering W503 and W504 at the same time, you want `extend-ignore` -- remove the arg here, set `noqa` on the individual lines, and don't change the `all(...)` it was correct how it was

but in reality, please do not ignore these, use noqa instead on the specific errors